src/client/ws/Gateway.ts

```diff
@@ -1,9 +1,7 @@
-import { Guild } from '#structures/Guild';
-import { Message } from '#structures/Message';
-import { User } from '#structures/User';
-import { log } from '#utils/logger';
+import { log } from 'console';
```

Comment:
```suggestion
import { log } from '../../util/logger';
```

Reply:
it's always been console I never changed that
![image](https://user-images.githubusercontent.com/69066026/164182345-34fb507e-d456-4ea4-96db-d93aa28b585a.png)